lib/hypernova/controller_helpers.rb

```diff
@@ -88,7 +88,7 @@ def hypernova_batch_after
           result = @hypernova_batch.submit!
           on_success(result, hash)
         rescue StandardError => e
-          on_error(e)
+          on_error(e, {}, hash)
```

Comment:
@ljharb in our plugin, this is how we define the `on_error` method currently

```
def on_error(error, job = {})
```
which supports [this call](https://github.com/airbnb/hypernova-ruby/blob/e6c6553a4838203f4cdd62368ba1c146f632233d/lib/hypernova/request_service.rb#L12). I was going to change it to...
```
def on_error(error, job = {}, hash = nil)
```

... but it is kind of weird. Any suggestion how to improve this?